includes/class-scd-ext-access-control.php

```diff
@@ -76,7 +76,11 @@ public function is_lesson_access_blocked( $lesson_id ) {
 		}
 
 		// Check if user has started the course.
-		$user_started_course = Sensei_Utils::user_started_course( $lesson_course_id, get_current_user_id() );
+		if ( Sensei_Content_Drip::instance()->is_legacy_enrolment() ) {
+			$user_started_course = Sensei_Utils::user_started_course( $lesson_course_id, get_current_user_id() );
```

Comment:
I wonder if it would make more sense to change the vars and comments (below) to match enrolment.